static/app/components/compactSelect/control.tsx

```diff
@@ -444,6 +444,8 @@ export function Control({
       if (e.key === 'ArrowDown' || e.key === 'ArrowUp') {
         e.preventDefault(); // Prevent scroll
         overlayState.open();
+      } else {
+        e.continuePropagation();
```

Comment:
Without continuing propagation, arrow left/right while the op trigger was focused wouldn't do anything. We probably should have had this here already, I don't foresee any issues with adding it.